### ISSUE TITLE
the fine-tuning of language in the content section

### ISSUE THREAD

**Issue by num3num** [poloclub/unitable#21]
Unitable is a powerful recognition tool, but I want to train table content recognition that supports other languages. Have any good suggestions or opinions?

**Comment by ShengYun-Peng** [comment]
I would suggest finetuning the OCR branch with the targeted language and UniTable should work out-of-the-box.

**Comment by num3num** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by num3num** [comment]
In the recognition of the bbox section, there may be a large amount of text or gaps in a single bbox, which can lead to content loss or misalignment. Do you have any good suggestions for this situation? What model or debugging method is called for pre training or fine-tuning of unitable_1arge_bbox.pt?

**Comment by Anananana1568** [comment]
> I would suggest finetuning the OCR branch with the targeted language and UniTable should work out-of-the-box.

请问您在微调的时候更改了哪些东西，有更改tokenizer嘛
